view/KKanghh/src/types/weatherData.ts

```diff
@@ -0,0 +1,4 @@
+export interface weatherData {
```

Comment:
```suggestion
export interface WeatherData {
```
- https://basarat.gitbook.io/typescript/styleguide#interface
- https://google.github.io/styleguide/tsguide.html#identifiers
타입의 이름은 통상적으로 PascalCase or UpperCamelCase로 적는 것이 좋습니다. :)

Reply:
감사합니다!!